apps/ledger-live-mobile/src/screens/Settings/Help/index.tsx

```diff
@@ -18,7 +18,6 @@ export default function HelpSettings() {
     <SettingsNavigationScrollView>
       <TrackScreen category="Settings" name="Help" />
       <LedgerSupportRow />
-      <ConfigureDeviceRow />
       <ExportLogsRow />
```

Comment:
You forgot to remove the `import ConfigureDeviceRow from "./ConfigureDeviceRow";` ;)

Reply:
Fixed